core/domain/rights_manager.py

```diff
@@ -222,6 +289,20 @@ def update_activity_first_published_msec(
 
 
 def create_new_exploration_rights(exploration_id, committer_id):
+    """Creates new exploration rights and save to the datastore.
+
+    Creates a default activity rights object for the exploration.
+    Creates the exploration object assigning owners, editors, viewers,
```

Comment:
I don't think this creates an exploration object. Also, exploration_rights is transient here.

Instead, perhaps something like this:

    Creates an ExplorationRightsModel instance for this exploration and saves it to the datastore. Subscribes the committer to the new exploration.